### ISSUE TITLE
Supraclavicular nerve distribution to findings

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10942]
**Preferred term label:**
Supraclavicular nerve distribution to findings

**Synonyms**

**Definition (free text, please give PubMed ID)**

Localization of symptoms such as numbness or paresthesias on the skin over the upper chest, specifically above the clavicle and over the lateral aspect of the neck

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Clavicle shaft fracture

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001169
- label: Supraclavicular nerve distribution
- parents: [Spatial pattern (HP:0012836)](https://hpo.jax.org/app/browse/term/HP:0012836)
- pmid: 
- def: Localization of symptoms such as numbness or paresthesias on the skin over the upper chest, specifically above the clavicle and over the lateral aspect of the neck, corresponding to the distribution of the supraclavicular nerve.
- comment: May be observed with clavicle shaft fracture.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10942
- synonyms:
